pkg/controller/replicaset/replica_set.go

```diff
@@ -84,7 +92,7 @@ type ReplicaSetController struct {
 	// A store of pods, populated by the podController
 	podStore cache.StoreToPodLister
 	// Watches changes to all pods
-	podController *framework.Controller
+	podController framework.ControllerInterface
```

Comment:
I copied it from ReplicationController which is mostly the same code as in ReplicaSet here.

I can remove it, but if you are ok with it, I would prefer doing this in the next PR, which this PR is blocking.
Does it sound ok?